labelImg.py

```diff
@@ -135,6 +135,16 @@ def __init__(self, defaultFilename=None, defaultPrefdefClassFile=None, defaultSa
         self.editButton = QToolButton()
         self.editButton.setToolButtonStyle(Qt.ToolButtonTextBesideIcon)
 
+
+        # *
+        # * dhzs 2017-12-2 add copy button
+        # *
+        self.copy_prev_button = QPushButton('复制上图')
```

Comment:
It is a nice feature. But could you change strings by getting it string id https://github.com/tzutalin/labelImg/blob/e1c67fc0c3e961e549b421e7c4ece13d4d259d4d/labelImg.py#L88